content/boards/unsupported/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Unsupported
+description: Details on unsupported boards and the issues with them.
+weight: 99
+---
+Many boards have similar names to MobiFlight [supported boards](../), but differ enough to not be supported
```

Comment:
I meant to put a style guide in place for this, but after going back and forth in many page edits I found it clearer to use absolute references to pages instead of relative. This should be `/boards` for example. Same applies to the rest of the files, `/boards/arduino-mega-2560-pro-mini`.

This was really apparent to me when attempting to link across document sections. Seeing the link is going to `/devices/led` was way better than `../../devices/led`.